tests/pass-dep/tokio/mpsc.rs

```diff
@@ -0,0 +1,20 @@
+//@only-target-x86_64-unknown-linux: support for tokio only on linux and x86
```

Comment:
This won't run the test on 32bit x86. Is that deliberate?

Reply:
Good point, it should be allowed on all linux target.